Index.html

```diff
@@ -24,17 +24,17 @@
 
     <section>
       <article>
-        <a href="academichallfire.html"
+        <a href="academichallfire.html">
         <header>
           <H2>The Academic Hall Fire</H2>
-        </header>
+        </header> 
         <p>On January 9, 1892, Academic Hall, the institution's main building, burned in a fire that completely gutted the building, leaving little more standing than six stone Ionic columns. Under the administration of Missouri Governor David R. Francis, the university was rebuilt, with additions that shaped the modern institution. On January 9, 1892, Academic Hall, the institution's main building, burned in a fire that completely gutted the building, leaving little more standing than six stone Ionic columns. Under the administration of Missouri Governor David R. Francis, the university was rebuilt, with additions that shaped the modern institution. (Wikipedia)</p>
          <img src="IMG/academic_hall_fire.jpg" alt="Picture of the Academic Hall fire" height="200" width="200">
        </a>
       </article>
 
       <article>
-         <a href="historyofdowntown.html"
+         <a href="historyofdowntown.html">
            <header>
```

Comment:
So throughout here, you have a header tag wrapping around your h2 elements -- this is extra code and overly verbose, since this isn't part of a section or article but is essentially functioning as a summary.